packages/widget/src/hooks/use-balances-by-chain.ts

```diff
@@ -27,7 +27,7 @@ export function useBalancesByChain({
   chain,
   assets,
   enabled = true,
-}: Args) {
+}: Args): UseQueryResult<Record<string, string>> {
```

Comment:
it seems like there should be a better type here, but `Record<string, string>` is explicitly used by a `reduce` below.